src/script/calling/CallingRepository.ts

```diff
@@ -664,8 +664,9 @@ export class CallingRepository {
     }
   };
 
-  async answerCall(call: Call, callType: CALL_TYPE): Promise<void> {
+  async answerCall(call: Call, callType?: CALL_TYPE): Promise<void> {
     try {
+      callType = callType ?? call.getSelfParticipant().sharesCamera() ? call.initialType : CALL_TYPE.NORMAL;
```

Comment:
```suggestion
      callType ??= call.getSelfParticipant().sharesCamera() ? call.initialType : CALL_TYPE.NORMAL;
```

Reply:
never seen this syntax.. interesting!!

Reply:
See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html :slightly_smiling_face: